packages/flutter_tools/lib/src/linux/linux_doctor.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/version.dart';
+
+import '../base/io.dart';
+import '../base/process_manager.dart';
+import '../doctor.dart';
+
+/// A validator that checks for GCC and the correct gtk version.
+class LinuxDoctorValidator extends DoctorValidator {
+  LinuxDoctorValidator() : super('Linux toolchain - develop for Linux desktop applications');
```

Comment:
s/desktop applications/desktop/, since you're developing applications for Linux desktop, not developing something for Linux desktop applications.